### DISCUSSION TITLE
New possible way to bypass momo?

### DISCUSSION THREAD

**Discussion by alan42521** [xfqwdsj/IAmNotADeveloper#82] [Ideas]
I found that after adding "System Framework" to the LSPosed scope in another recently released module called Imnotadeveloper, momo no longer shows the debugging mode enabled notification, although there are still other issues.
How does it achieve it?

**Comment by xfqwdsj** [comment]
You're right. We can hook the system to make it provide tampered results to the apps, which cannot be detected.

Specifically, what we are currently doing is to modify the application so that it “confuses itself”. This modification behavior can be detected. However, the other (or "new") method makes the system directly provide the modified result to the apps. The apps can only choose to believe the result given by the system because it cannot know whether the result has been tampered with.

**Reply by alan42521** [reply]
So could the module use the same way to trick momo?

**Reply by xfqwdsj** [reply]
Since they can do it, so can we. However, using this method requires us to maintain a list of applications that should be tampered with because the LSPosed scope can no longer control which applications should be tampered with. Moreover, I am currently busy with my studies and do not have the time to do this for now.
